prow/cmd/hook/main.go

```diff
@@ -173,12 +174,23 @@ func main() {
 
 	pluginAgent := &plugins.PluginAgent{}
 
+	ownersDirBlacklistConf := configAgent.Config().OwnersDirBlacklist
```

Comment:
This means that we lose the ability to dynamically update the blacklist iow. every change in it requires a restart of hook. I am fine with fixing this in a follow-up, can you open an issue to track it?